controllers/nstemplatetier/nstemplatetier_controller_test.go

```diff
@@ -395,6 +395,56 @@ func TestUpdateNSTemplateTier(t *testing.T) {
 				})
 		})
 	})
+
+	t.Run("when updating one tiertemplate the revisions field should be cleaned up from old entries", func(t *testing.T) {
```

Comment:
Minor, should we also check the case, if we create new ttr, the status.revisions is updated , if its not too complicated to write?

Reply:
The TTRs are created by the nstemplatetier_controller, and the status.revisions update is being checked in most of the tests using :

```
tiertest.AssertThatNSTemplateTier(t, "base1ns", cl).
					HasStatusTierTemplateRevisions(tierTemplatesRefs)
``` 

see for example : https://github.com/codeready-toolchain/host-operator/blob/508b798e82cbdd5b706fde35afbcbb671ffa130a/controllers/nstemplatetier/nstemplatetier_controller_test.go#L129-L140

is this what you were looking for ?

Reply:
No i mean with respect to the deletion ,
i was thinking maybe https://github.com/codeready-toolchain/host-operator/pull/1136/files#diff-b4dc433d5162cf28853736dd0fa3f879568b9c587546af8b321c9f304664d27cR116
if this condition can somehow be checked in unit test cases

Reply:
i see matous's comment of not requiring ttrcreated variable.. if thats the case.. you dont require the case i am talking about

Reply:
this is already covered in the unit-test. the test replaces TierTemplate reference with a new one, which means that it will trigger creation of new TTR CR, plus i twill require deletion of the old entries from the revisions map.